.github/workflows/integration.yml

```diff
@@ -31,7 +31,7 @@ jobs:
         mvn -B org.apache.maven.plugins:maven-dependency-plugin:3.1.2:go-offline -f fhir-parent -DexcludeReactor=true -Dmaven.wagon.http.retryHandler.count=3
         mvn -B org.apache.maven.plugins:maven-dependency-plugin:3.1.2:resolve-plugins -f fhir-parent -DexcludeReactor=true -Dmaven.wagon.http.retryHandler.count=3
         mvn -B install --file fhir-parent -DskipTests -P include-fhir-igs,integration --no-transfer-progress -Dmaven.wagon.http.retryHandler.count=3
-        mvn -B dockerfile:build -f fhir-install --no-transfer-progress -Dmaven.wagon.http.retryHandler.count=3 -Ddocker.buildArg.VERBOSE=false
+        docker build fhir-install --build-arg VERBOSE=false -t linuxforhealth/fhir-server
```

Comment:
Is the docker build process migrating from maven to the docker CLI? I didn't know if there were relevant portions of the POM(s) which could be removed. I did see this [snippet](https://github.com/LinuxForHealth/FHIR/pull/3886/files#diff-0b51d62d390dc0b76f0597ee8788fbb662104fbad227ba8aa5bd43d494fbe3f0L134) but of course I'm not familiar with the overall context, pun intended :)

Reply:
It's a step in that direction.  Today we have pretty even a mix of `docker build` and `mvn dockerfile:build` in our various workflows.  After this round of changes, we'll use the `docker build` flavor almost exclusively and we'll build to a local image named `linuxforhealth/fhir-server`.

The one place I left the maven-based docker build is in our release workflow.  In that one case we build to `ghcr.io/linuxforhealth/fhir-server`.  See https://github.com/LinuxForHealth/FHIR/blob/c28cb62b3b3188dad33f3086033c8bf5fa279048/build/release/bin/30_release/1_docker.sh#L29-L32

But we don't really *need* to use the dockerfile-maven-plugin there either.  Do you think we should change to use the docker cli there as well?

Additionally we could use the `ghcr.io/linuxforhealth/fhir-server` flavor all over...I just felt that keeping the registry prefix out of the name in the other workflows might help make it more clear that its built locally (and won't fall back to pulling from the registry if we ever fail to build it).